src/CarbonAware.WebApi/test/integrationTests/SciScoreControllerTests.cs

```diff
@@ -22,11 +22,13 @@ public class SciScoreControllerTests : IntegrationTestingBase
 
 
     [TestCase("2022-1-1T04:05:06Z", "2022-1-2T04:05:06Z", "eastus", HttpStatusCode.OK)]
-    [TestCase("2021-1-1", "2022-1-2", "westus", HttpStatusCode.OK)]
-    public async Task SCI_AcceptsValidData_ReturnsContent(DateTimeOffset start, DateTimeOffset end, string location, HttpStatusCode expectedCode)
+    [TestCase("2021-11-18", "2022-1-2", "westus", HttpStatusCode.OK)]
+    public async Task SCI_AcceptsValidData_ReturnsContent(string start, string end, string location, HttpStatusCode expectedCode)
     {
-        _dataSourceMocker.SetupDataMock(start, end, location);
-        string timeInterval = start.ToUniversalTime().ToString("O") + "/" + end.ToUniversalTime().ToString("O");
+        var startTime = DateTimeOffset.Parse(start);
+        var endTime = DateTimeOffset.Parse(end);
```

Comment:
I would check this works in a non-dev container environment where by default it's not UTC, I'm pretty sure `DateTimeOffset.Parse(...)` will parse the date into local time so if these tests are ever run in non-UTC environments they might fail

Reply:
the tests all passed locally in VisStudio, so think we're ok

Reply:
Confirmed.  It all checks out.